packages/now-cli/package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "preinstall": "node ./scripts/preinstall.js",
-    "test-unit": "nyc ava test/unit.js test/dev-builder.unit.js test/dev-router.unit.js test/dev-server.unit.js --serial --fail-fast --verbose",
+    "test-unit": "nyc ava test/unit.js test/dev-builder.unit.js test/dev-router.unit.js test/dev-server.unit.js test/dev-validate.unit.js --serial --fail-fast --verbose",
```

Comment:
@TooTallNate I added a unit test file for this but I realized that we already have some integration tests for error messages like this. Should I move all my new tests to dev fixtures instead of unit tests?

Reply:
Good question… it makes sense to me to have comprehensive unit tests for many scenarios, but only a few integration tests to make sure the code path(s) to print the validation errors in the CLI are working as expected.